installation/network-access.md

```diff
@@ -0,0 +1,20 @@
+---
+description: >-
+  The following document covers the network access requirements for the Edge Delta Agent.
+---
+
+# Agent Network Access Requirements
+
+Edge Delta Agent can be deployed in restricted network environments where outgoing network traffic to the Internet is strictly moderated.
+In such an environment network policies should be updated by enabling outgoing access to these destinations and bypassing HTTP/HTTPS proxy.
+
+## Edge Delta Backend
+
+The agent requires outbound network access to Edge Delta central systems to be fully functionally. These destinations are account specific.
+If you have restricted outbound network policy contact [support@edgedelta.com](mailto:support@edgedelta.com) to the get list of destination addresses for your account.
```

Comment:
do you think we can share the outbound network destinations here (from the doc)? I guess it should be OK to share

Reply:
I am not sure, my initial intent was to put them here, we also have a backend where we might assign different org resources.  Not sharing does not provide any security technically of course but it is like putting you email on your home page, let's discuss in standup.

Reply:
In general having things in documentation is good because documentation scales better than our sales/support team in terms of responding to customer questions. That's why I asked maybe we just drop the information here and don't worry about these inquiries but I guess outbound traffic restrictions is not that common so it's OK to not share here initially. LGTM as is to me